tests/state_migration_test.go

```diff
@@ -22,17 +22,19 @@ import (
 	"github.com/klaytn/klaytn/node/cn"
 	"github.com/klaytn/klaytn/storage/database"
 	"github.com/stretchr/testify/assert"
+	"github.com/syndtr/goleveldb/leveldb"
 	"math/big"
 	"os"
+	"path/filepath"
 	"strconv"
 	"testing"
 	"time"
 )
 
-// continues occurrence of state trie migration and node restart must success
+// continuous occurrence of state trie migration and node restart must success
 func TestMigration_ContinuesRestartAndMigration(t *testing.T) {
 	fullNode, node, validator, chainID, workspace, richAccount, _, _ := newSimpleBlockchain(t, 10)
-	defer os.RemoveAll(workspace)
+	//defer os.RemoveAll(workspace)
```

Comment:
Why did you comment here?

Reply:
Sorry, I might commented out while testing.